jobs/germany-crowd-forecast-data-update.sh

```diff
@@ -0,0 +1,16 @@
+#!bin/bash
+
+# check repository is up to date
+docker exec -w /home/rstudio/covid-german-forecasts forecast git pull -Xours
```

Comment:
There is quite a bit of repetition here. If putting into a shared `covid-germany` folder could have an `update-dependencies.sh` script to pull and install packages and then call that in each of these jobs rather than writing it again multiple times.